README.md

```diff
@@ -24,11 +24,11 @@ In order to convert your EL8 operating system to AlmaLinux do the following:
    ```shell
    # check release file
    $ cat /etc/redhat-release 
-   AlmaLinux release 8.3 (Purple Manul)
+   AlmaLinux release 8.4 (Purple Manul)
```

Comment:
Please change codename to Electric Cheetah

Reply:
```suggestion
   AlmaLinux release 8.4 (Electric Cheetah)
```